src/Engine/Engine.cpp

```diff
@@ -538,33 +514,43 @@ void Engine::LogEngineBuildInfo() {
 }
 
 //----- (0044EA5E) --------------------------------------------------------
-void Engine::PickMouse(float fPickDepth, unsigned int uMouseX,
-                       unsigned int uMouseY, bool bOutline,
-                       Vis_SelectionFilter *sprite_filter,
-                       Vis_SelectionFilter *face_filter) {
+Vis_PIDAndDepth Engine::PickMouse(float fPickDepth, unsigned int uMouseX, unsigned int uMouseY,
+                                  Vis_SelectionFilter *sprite_filter, Vis_SelectionFilter *face_filter) {
     if (uMouseX >= (signed int)pViewport->uScreen_TL_X &&
         uMouseX <= (signed int)pViewport->uScreen_BR_X &&
         uMouseY >= (signed int)pViewport->uScreen_TL_Y &&
         uMouseY <= (signed int)pViewport->uScreen_BR_Y) {
-        vis->PickMouse(fPickDepth, uMouseX, uMouseY, sprite_filter, face_filter);
-
-        if (bOutline)
-            OutlineSelection();
```

Comment:
Why outline is dropped?

Reply:
Would it ever be a useful debug tool to draw a box around hovered sprites?

Reply:
Would be easy to implement if we ever needed it. Happy to drop.

Reply:
It wasn't really used. The polygon highlighting that we have goes through another codepath.